src/components/Footer.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import logoGps from "../images/logoGps.png";
+import logoNote from "../images/logoNote.png";
+import styled from "styled-components";
+
+const FooterStyled = styled.div`
+  background-color: #242424;
+  width: 100vw;
+  height: 10vh;
+  display: flex;
+  justify-content: space-around;
+`;
+
+const ImageStyled = styled.img`
+  height: 90%;
+  width: auto;
+  margin: auto;
+`;
+
+function Footer() {
+  return (
+    <FooterStyled>
+      <ImageStyled src={logoGps} alt="gps"></ImageStyled>
```

Comment:
Replace with `<ImageStyled src={logoGps} alt="gps" />`
(And do this for every image on the project)